src/router/Router.js

```diff
@@ -164,6 +167,18 @@ export default class Router extends DisposableBase {
             return this._observeEventsUsingConventions(modelId, object, methodPrefix);
         }
     }
+    addOnErrorHandler(handler) {
+        this._onErrorHandlers.push(handler);
```

Comment:
I forgot about this.  Shall I add another PR?

On 24 February 2016 at 14:41, Keith Woods <notifications@github.com> wrote:

> In src/router/Router.js
> <https://github.com/esp/esp-js/pull/71#discussion_r53938949>:
>
> > @@ -164,6 +167,18 @@ export default class Router extends DisposableBase {
> >              return this._observeEventsUsingConventions(modelId, object, methodPrefix);
> >          }
> >      }
> > +    addOnErrorHandler(handler) {
> > +        this._onErrorHandlers.push(handler);
>
> should blow up if this isn't a function, i.e. simple
> Guard.isFunction(handler).
>
> Looks like most other public methods have similar (except runAction which
> should).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/esp/esp-js/pull/71/files#r53938949>.
>